pkg/daemon/ceph/osd/volume.go

```diff
@@ -146,7 +146,7 @@ func updateLVMConfig(context *clusterd.Context) error {
 	output = bytes.Replace(output, []byte("use_lvmetad = 1"), []byte("use_lvmetad = 0"), 1)
 	output = bytes.Replace(output, []byte("obtain_device_list_from_udev = 1"), []byte("obtain_device_list_from_udev = 0"), 1)
 	output = bytes.Replace(output, []byte(`scan = [ "/dev" ]`), []byte(`scan = [ "/dev", "/mnt" ]`), 1)
-	output = bytes.Replace(output, []byte(`# filter = [ "a|.*/|" ]`), []byte(`filter = [ "a|^/mnt/.*| r|.*/|" ]`), 1)
+	output = bytes.Replace(output, []byte(`# filter = [ "a|.*/|" ]`), []byte(`filter = [ "a|^/mnt/.*|", "r|.*|" ]`), 1)
```

Comment:
Was there a bug around this? Or it's just making an update according to the pattern in the documentation?

Reply:
It's hard to tell at this point if the rule was working or not.

Reply:
@leseb @sp98 Do we even need the reject rule? Or can we keep it simple and just add /mnt to the rule?

Reply:
The reject rule *could* fix the issue we see with loop devices so yes I think the rule makes or could be changed to reject loop dev but the inclusion rule looks safer.

Reply:
I just tested and actually, that seems to be solving the issue! So I suppose the regex was incorrect. I'm going to do some further testing to verify that.